syntax = "proto3";

package matching;

// Сервис для поиска совпадений
service Matching {
  // Метод для поиска совпадений для пользователя
  rpc FindMatches (FindMatchesRequest) returns (FindMatchesResponse);
}

// Запрос на поиск совпадений
message FindMatchesRequest {
  string user_id = 1; // ID пользователя, для которого ищем совпадения
}

// Ответ со списком совпадений
message FindMatchesResponse {
  repeated string matched_user_ids = 1; // Список ID пользователей, которые совпали
}